pandas/tseries/tests/test_base.py

```diff
@@ -682,6 +781,52 @@ def test_minmax(self):
             obj = TimedeltaIndex([pd.NaT, pd.NaT, pd.NaT])
             self.assertTrue(pd.isnull(getattr(obj, op)()))
 
+    def test_numpy_minmax(self):
+        dr = pd.date_range(start='2016-01-15', end='2016-01-20')
+        td = TimedeltaIndex(np.asarray(dr))
+
+        self.assertEqual(np.min(td), Timedelta('16815 days'))
+        self.assertEqual(np.max(td), Timedelta('16820 days'))
+
+        errmsg = "the 'out' parameter is not supported"
+        tm.assertRaisesRegexp(ValueError, errmsg, np.min, td, out=0)
+        tm.assertRaisesRegexp(ValueError, errmsg, np.max, td, out=0)
+
+        self.assertEqual(np.argmin(td), 0)
+        self.assertEqual(np.argmax(td), 5)
+
+        # the 'out' parameter was introduced into numpy's
+        # arg(min/max) starting in version 1.10.0
+        if not _np_version_under1p10:
+            errmsg = "the 'out' parameter is not supported"
+            tm.assertRaisesRegexp(ValueError, errmsg, np.argmin, td, out=0)
+            tm.assertRaisesRegexp(ValueError, errmsg, np.argmax, td, out=0)
+
+    def test_round(self):
+        rng = pd.date_range(start='2016-01-01', periods=5, freq='30Min')
```

Comment:
well very hard to see what you are doing here as the comments obsure things, so no idea